ui/app/components/server-agent-row.js

```diff
@@ -24,11 +24,12 @@ export default Component.extend({
 
     const router = this.get('router');
     const targetURL = router.generate('servers.server', this.get('agent'));
-    const currentURL = `${router.get('rootURL').slice(0, -1)}${router
-      .get('currentURL')
-      .split('?')[0]}`;
+    const currentURL = `${router.get('rootURL').slice(0, -1)}${
+      router.get('currentURL').split('?')[0]
+    }`;
 
-    return currentURL === targetURL;
+    // Account for potential URI encoding
+    return currentURL.replace(/%40/g, '@') === targetURL.replace(/%40/g, '@');
   }),
 
   click() {
```

Comment:
What's this for?

Reply:
This is a realish bug that I found thanks to `visitable`. Since `visitable` encodes paths, the active server wasn't getting the `is-active` class. 

I say "realish" because browsers handle an `@` in the url just fine, but since both forms are acceptable, they should both be checked against.